circle.yml

```diff
@@ -0,0 +1,68 @@
+machine:
+  environment:
+    # We need to set this variable to let Anaconda take precedence
+    PATH: "/home/ubuntu/miniconda/envs/circleenv/bin:/home/ubuntu/miniconda/bin:$PATH"
+    MNE_DATA: "/home/ubuntu/mne_data"
```

Comment:
ahh okay I must have misread it. @agramfort you okay wiht `MNE_DATA_PATH` instead of `MNE_DATA`? All of our other path names have `_PATH` at the end.

Also, you'll need to add it to the known keys in `utils.py`.